backend/api/v1/auth_service.go

```diff
@@ -215,15 +208,7 @@ func (s *AuthService) CreateUser(ctx context.Context, request *v1pb.CreateUserRe
 				}
 				userMessage.Roles = append(userMessage.Roles, api.Role(roleID))
 			}
-		}
-	}
-	// If multiple roles are specified, checks if the current user is workspace admin.
-	if len(userMessage.Roles) > 1 {
-		user, ok := ctx.Value(common.UserContextKey).(*store.UserMessage)
-		if !ok {
-			return nil, status.Error(codes.PermissionDenied, "user not found in context")
-		}
-		if !slices.Contains(user.Roles, api.WorkspaceAdmin) {
+		} else if len(request.User.Roles) > 0 {
```

Comment:
Could `request.User.Roles` be `[workspaceMember]`?

Reply:
No, we changed the backend & frontend before, the `workspaceMember` will be the default role.
But this may affect API users

Reply:
Also wanna to get rid of Roles field from User proto, too. At least, not creating user with roles or mutating user roles

Reply:
So user creation should be: 1. create user 2. then update the workspace IAM policy?
This need to refactor the frontend, I'd like to do it in following PR

Reply:
Yes, please. Wasn't expecting to do them all in the same PR.